app/controllers/api/v1/accounts/deliveries_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Api::V1::Accounts::DeliveriesController < Api::BaseController
```

Comment:
Hm… why is it in `/api/v1/accounts/`? More generally, I wonder if all of those new APIs could be in something like `/api/v1/e2ee/` or `/api/v1/crypto/`.